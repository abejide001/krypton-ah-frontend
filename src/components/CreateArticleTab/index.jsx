import React from 'react';
import { Tab, Container } from 'semantic-ui-react';
import Editor from '../Editor';

const panes = [
  {
    menuItem: 'Text Entry',
    render: function TabPane() {
      return (
        <Tab.Pane attached={false}>
          <Editor />
        </Tab.Pane>
      );
    }
  }
];

function CreateArticleTab() {
  return (
    <Container>
      <Tab menu={{ secondary: true }} panes={panes} />
    </Container>
  );
}


export default CreateArticleTab;
