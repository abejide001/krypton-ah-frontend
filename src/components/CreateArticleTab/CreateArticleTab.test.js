import React from 'react';
import { shallow } from 'enzyme';
import CreateArticleTab from '.';

describe('ArticleTab Component', () => {
  it('should rendered ArticleTab component without crashing', () => {
    const wrapper = shallow(<CreateArticleTab />);
    expect(wrapper).toMatchSnapshot();
  });
});
