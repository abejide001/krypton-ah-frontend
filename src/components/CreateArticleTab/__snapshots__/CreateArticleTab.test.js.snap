// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleTab Component should rendered ArticleTab component without crashing 1`] = `
<Container>
  <Tab
    grid={
      Object {
        "paneWidth": 12,
        "tabWidth": 4,
      }
    }
    menu={
      Object {
        "secondary": true,
      }
    }
    panes={
      Array [
        Object {
          "menuItem": "Text Entry",
          "render": [Function],
        },
      ]
    }
    renderActiveOnly={true}
  />
</Container>
`;
